include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -969,6 +969,56 @@ namespace internal
 
   class MGTwoLevelTransferImplementation
   {
+    template <int dim, typename Number>
+    static std::shared_ptr<const Utilities::MPI::Partitioner>
+    create_coarse_partitioner(
+      const DoFHandler<dim> &                  dof_handler_coarse,
+      const dealii::AffineConstraints<Number> &constraint_coarse,
```

Comment:
Not related to this PR, but it seems that this file uses the term `constraint_coarse`, whereas most other places would spell this as `constraints_coarse` (with the plural form).

Reply:
Done!